crates/frontend/src/components/table.rs

```diff
@@ -19,18 +63,27 @@ pub fn table(
 ) -> impl IntoView {
     let pagination_props = StoredValue::new(pagination);
     let container_style = format!("{} overflow-x-auto", class);
+    let get_sticky_position_classes = |index: usize| {
+        match index {
+        0 => "sticky left-20 z-20 bg-inherit after:content-[''] after:absolute after:right-0 after:top-0 after:bottom-[-1px] after:w-[5px] after:bg-[linear-gradient(90deg,rgba(0,0,0,0.1)_0%,rgba(0,0,0,0)_100%)]",
+        _ => "",
+    }
+    };
```

Comment:
nitpick: would it be better to do:
```suggestion
    // this constant is at the top somewhere
    const STICKY_CLASS_FOR_COL = "sticky left-20 z-20 bg-inherit after:content-[''] after:absolute after:right-0 after:top-0 after:bottom-[-1px] after:w-[5px] after:bg-[linear-gradient(90deg,rgba(0,0,0,0.1)_0%,rgba(0,0,0,0)_100%)]";
    // then use it where ever needed
    let get_sticky_position_classes = if index == 0 { STICKY_CLASS_FOR_COL } else { "" };
    };
```